ConnectorAutocadCivil/ConnectorAutocadCivil/Entry/SpeckleAutocadCommand.cs

```diff
@@ -114,7 +115,10 @@ public static void SpeckleCommunity()
         true
       );
     }
-    catch { }
+    catch (System.Exception ex)
```

Comment:
Are these all top-level exceptions? If we didn't catch this, would the host app have to handle it (and possibly crash?) if so, all good - we have a ticket for better handling for TL-exceptions